src/ui/view/governance/details/details.scss

```diff
@@ -173,6 +173,27 @@
   }
 }
 
+.okp4-dataverse-portal-governance-details-modal-top-element {
+  display: flex;
+  gap: 6px;
+  align-items: center;
+
+  h1 {
```

Comment:
We might prefer to not relying on HTML tag to define style. In this case it will apply the style on every h1 from modal top element. Which is probably not what we want. Defining a class for your title is more robust here.

Reply:
Ah yes you are absolutely right!